patches/@clerk__astro@1.5.2.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/package.json b/package.json
+index ecf86cb4c8cf86d6fc7d6455d9dbca69ff8b0eea..075f1f5e7fe115dfb9bda50d1ccfe4eb616c15f3 100644
+--- a/package.json
++++ b/package.json
+@@ -29,9 +29,9 @@
+   "imports": {
+     "#async-local-storage": {
+       "workerd": "./dist/async-local-storage.server.js",
+-      "browser": "./dist/async-local-storage.client.js",
+       "node": "./dist/async-local-storage.server.js",
+-      "default": "./dist/async-local-storage.server.js"
++      "default": "./dist/async-local-storage.server.js",
++      "browser": "./dist/async-local-storage.client.js"
```

Comment:
Cloudflare always resolves to the `browser` condition here. Moved it to bottom for now to make it work